content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,276 @@
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to save the
+model and add it to the model registry. Open the training script `src/train.py`
+in our example repository and you will see the following code under the
+`with Live(...)` context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in the Studio registry (other artifact types will not) and the rest of the
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+The training script runs as a part of a dvc pipeline
+[we prepared](/doc/start/experiments/experiment-pipelines) in the Experiment
+Management guide, so now we just call `dvc exp run` to run the experiment.
+
+We now commit the result to git (and push it to our git remote) and the new
+model will show up in the model registry in Studio. We will also push the
+resulting artifact to our DVC Remote by calling `dvc push`.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provided in the call and edits the `dvc.yaml` file which will now contain the
+following lines
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your git remote it is parsed by Studio and
+it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+You should see something like the following picture in the Studio Model
+Registry.
+
+![Newly added model in the Model Registry](/img/mr-newly-added-model.png)
+
+To get acquainted with how DVC stores and shares data, see our
+[Get Started guide on Data Versioning](/doc/start/data-management/data-versioning).
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the Studio's graphical user interface to add
+models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## DVC Model registry overview
```

Comment:
Should we link more liberally to the demo project here? Users can't really act on this themselves right now without making a personal fork, setting up their own remote, etc., but we can show them exactly what the model registry looks like for this project.

Reply:
A fork won't copy the tags though, so I think this might actually be a decent place to start: you have the model and can see the code used to generate it, but you don't have anything registered yet.

Reply:
Yeah, I guess that's right, yo don't have to fork it with tags (and by default you don't). Then I guess we don't even have to mention `dvc exp run` and running the experiments. We can just assume those have been run already and ask users to call `dvc pull` to pull them to their form from our public registry. Then for the model registry actions they don't need to `dvc push` anything and so they can follow along.

Reply:
I will make one commit where we keep the `dvc exp run` and then another one with just pulling everything ready made. Then we can see which is better.